lib/Connection.php

```diff
@@ -118,6 +150,8 @@ public function send(array $strings): Promise
             yield $this->connect();
             yield $this->socket->write($payload);
         });
+
+        return $deferred->promise();
```

Comment:
@kelunik tests are passing but I'm not 100% sure if I have changed invoking `call` and returning `Deferred` properly. Any advice if it is wrong?